projects/tinkerbell/sandbox-local-registry/.env

```diff
@@ -0,0 +1,5 @@
+TINK_CLI_IMAGE="public.ecr.aws/l0g8r8j6/tinkerbell/tink/tink-cli:v0.6.0-eks-a-v0.0.0-dev-build.1536"
+TINK_SERVER_IMAGE="public.ecr.aws/l0g8r8j6/tinkerbell/tink/tink-server:v0.6.0-eks-a-v0.0.0-dev-build.1536"
+BOOTS_SERVER_IMAGE="public.ecr.aws/l0g8r8j6/tinkerbell/boots:v0.6.0-eks-a-v0.0.0-dev-build.1536"
+HEGEL_SERVER_IMAGE="public.ecr.aws/l0g8r8j6/tinkerbell/hegel:v0.6.0-eks-a-v0.0.0-dev-build.1536"
+PBNJ_IMAGE_URL="public.ecr.aws/l0g8r8j6/tinkerbell/pbnj:7275abd396e5fce9b1358e18fe6d935f6cb341af-eks-a-v0.0.0-dev-build.1536"
```

Comment:
can we point these to the later builds? I see the latest ones are tagged `v0.6.0-eks-a-v0.0.0-dev-build.2151`

Reply:
what about pbnj? it has a different tag?